Dashboard/app/js/lib/views/views.js

```diff
@@ -519,26 +519,19 @@ Ember.Handlebars.registerHelper('sgName', function (property) {
 });
 
 Ember.Handlebars.registerHelper('surveyPath', function (property) {
-  var formId = Ember.get(this, property), path = "", sgs = FLOW.projectControl.get('content'), survey = null;
-  if (sgs) {
-    sgs.forEach(function(item) {
-      var surveysList = item.get('surveyList');
-      if (item.get && surveysList && surveysList.indexOf(formId) > -1) {
-        survey = item;
-      }
-    });
-    if (survey) {
-      var ancestorIds = survey.get('ancestorIds')
-      for (var i = 0; i < ancestorIds.length; i++) {
-        if (ancestorIds[i] !== null && ancestorIds[i] !== 0) {
-          var level = FLOW.SurveyGroup.find(ancestorIds[i]);
-          if (level) {
-            path += (i > 0 ? " > ": "")+level.get('name');
-          }
+  var formId = Ember.get(this, property), path = "";
+  var survey  = FLOW.Survey.find(formId);
+  if (survey) {
+    var ancestorIds = survey.get('ancestorIds')
+    for (var i = 0; i < ancestorIds.length; i++) {
+      if (ancestorIds[i] !== null && ancestorIds[i] !== 0) {
+        var level = FLOW.SurveyGroup.find(ancestorIds[i]);
```

Comment:
We should probably use the terms that we actually want to use. I.e. what we call survey is a form and what we are calling level is a survey.